src/mapping/tests/RadialBasisFctMappingTest.cpp

```diff
@@ -187,25 +158,35 @@ BOOST_AUTO_TEST_CASE(DistributedConsistent2DV1)
   PRECICE_TEST(""_on(4_ranks).setupIntraComm());
   Multiquadrics fct(5.0);
 
-  MeshSpecification in{// Consistent mapping: The inMesh is communicated
-                       {-1, 0, {0, 0}, {1}},
-                       {-1, 0, {0, 1}, {2}},
-                       {-1, 1, {1, 0}, {3}},
-                       {-1, 1, {1, 1}, {4}},
-                       {-1, 2, {2, 0}, {5}},
-                       {-1, 2, {2, 1}, {6}},
-                       {-1, 3, {3, 0}, {7}},
-                       {-1, 3, {3, 1}, {8}}};
-  MeshSpecification out{// The outMesh is local, distributed among all ranks
-                        {0, -1, {0, 0}, {0}},
-                        {0, -1, {0, 1}, {0}},
-                        {1, -1, {1, 0}, {0}},
-                        {1, -1, {1, 1}, {0}},
-                        {2, -1, {2, 0}, {0}},
-                        {2, -1, {2, 1}, {0}},
-                        {3, -1, {3, 0}, {0}},
-                        {3, -1, {3, 1}, {0}}};
-
+  std::vector<VertexSpecification> inVertexList = {
+      {-1, 0, {0, 0}, {1}},
+      {-1, 0, {0, 1}, {2}},
+      {-1, 1, {1, 0}, {3}},
+      {-1, 1, {1, 1}, {4}},
+      {-1, 2, {2, 0}, {5}},
+      {-1, 2, {2, 1}, {6}},
+      {-1, 3, {3, 0}, {7}},
+      {-1, 3, {3, 1}, {8}}};
+
+  MeshSpecification in = {
+      inVertexList,
+      static_cast<int>(static_cast<int>(inVertexList.at(0).position.size())),
+      nullptr};
+
+  std::vector<VertexSpecification> outVertexList = {
```

Comment:
There is no reason not to directly construct the list:
```suggestion
  std::vector<VertexSpecification> outVertexList{
```